### ISSUE TITLE
react-ciscospark doesnt compile on windows 

### ISSUE THREAD

**Issue by arun3528** [webex/react-widgets#353]
I am trying to run the demo app on windows and get the following error 
![babel-error](https://user-images.githubusercontent.com/3895486/28176047-d73c189c-67aa-11e7-9e84-763ca3f560ee.PNG)

**Comment by pcmgs** [comment]
Has this issue been fixed? #440

**Comment by pgoldweic** [comment]
I've tried following the instructions for Windows but the build is definitely broken. First, npm was failing to rebuild 'node-gyp' -when running 'npm install'-  with an error about not finding the Python executable (which happened to be in place!). Eventually I got this part to work by following instructions online to install node-gyp on Windows (which unfortunately installed Python 2.7) . Then tried to run the next step ('npm run build') but it failed immediately with the following message:
     '.' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @ciscospark/react-ciscospark@0.1.235 build: `./scripts/build/index.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @ciscospark/react-ciscospark@0.1.235 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
